src/main/java/org/opentripplanner/routing/edgetype/StreetEdgeTraversal.java

```diff
@@ -243,14 +274,22 @@ public StateEditor doTraverse(RoutingRequest options, TraverseMode traverseMode)
                 // TODO: how to weight off of these values?
             }
 
+            if (options.wheelchairAccessible && curbRamp == CurbRamp.NO) {
```

Comment:
Oh, good catch.  I meant to refactor that into the curb ramp weighting for walkers and canes.